File: Easy/SumOfIntegersFromFile.py
# Eden Ghirmai, 3/05/14, www.codeeval.com
# Prints out the sum of integers read from a file.

import sys
test_cases = open(sys.argv[1], 'r')
sum = 0
for test in test_cases:
    if test:
    	sum += int(test)
print sum

test_cases.close()    	

File: Easy/SumOfIntegersFromFile.java
// Eden Ghirmai, 3/05/2014, www.codeeval.com
// Prints out the sum of integers read from a file.

import java.io.*;
import java.util.*; 

public class SumOfIntegersFromFile {
    public static void main(String[] args) throws FileNotFoundException {
		File file = new File(args[0]);
		Scanner in = new Scanner(file);
		
		int sum = 0; 
		while(in.hasNextLine()) {
			sum += Integer.parseInt(in.nextLine());
		}
		System.out.println(sum);
    }
}